src/components/inputs/SelectList.tsx

```diff
@@ -0,0 +1,47 @@
+import { ReactNode } from "react";
+import PopoverButton from "../buttons/PopoverButton";
+import CheckboxList, {LinksPosition} from "./CheckboxList";
+
+type Item = {
+    display: ReactNode
+    value: any
+  }
+  
+type SelectListProps = {
+    name?: string
+    items: Item[]
+    value: string[]
+    onChange: (value: string[]) => void
+    linksPosition?: LinksPosition
+    popoverCustomizations?: ReactNode
+    buttonDisplayContent: ReactNode
+    onClose?: unknown
```

Comment:
I think this was added to support updating the selection a user makes when the popover is closed? I wonder if we can do this differently.

This component can keep track of what the user has selected as component state. Instead of passing this component's `onChange` callback prop to `CheckboxList`, a custom callback can be passed to it that updates this component's state. Another custom callback, which calls this component's `onClose` prop with the selection kept as state, can be passed as the `onClose` prop for `PopoverButton`. This would obviate the need for this `onClose` prop.